test/lit-mobx.test.ts

```diff
@@ -50,20 +58,30 @@ describe('MobxLitElement', () => {
         const el = await fixture<TestMobxLitElement>(html`
             <test-mobx-lit-element></test-mobx-lit-element>
         `);
+        // we should see the updated value from firstUpdated
+        expect(el).shadowDom.to.equal(
```

Comment:
Nit: can we move these comments into the `expect`? This way we'll be able to see the comments in the code and the test output (when failing) to get a clearer understanding of any failures in the test output.
```suggestion
        expect(el, 'we should see the updated value from firstUpdated').shadowDom.to.equal(
```

Reply:
Good call, done in eb46001